administrator-manual/en/docker.rst

```diff
@@ -12,8 +12,8 @@ Docker is an open platform for developing, shipping, and running applications. D
 
 .. warning::
 
- Docker is customised to NethServer and the firewall layer. It means that the default bridge ``docker0`` is disabled and we use other bridge called ``aqua`` and ``aeria``.
```

Comment:
Instead of `NethServer` use the macro `|product|`.

Reply:
done